examples/with-solidstart/src/auth/Provider.tsx

```
@@ -0,0 +1,100 @@
+        }
+      },
+      { defer: true }
+    )
+  );
+
+  const logout = async () => {
+    try {
+      const { label } = onboard!.connectedWallet();
+      await onboard!.disconnectWallet({ label });
+    } finally {
+      return signOut();
```

Biome (2.1.2)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)